pkg/gcmanager/pod_informer.go

```diff
@@ -0,0 +1,278 @@
+// Copyright 2022 Authors of spidernet-io
+// SPDX-License-Identifier: Apache-2.0
+
+package gcmanager
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+
+	"github.com/spidernet-io/spiderpool/pkg/constant"
+)
+
+// registerPodInformer will set up k8s pod informer in circle
+func (s *SpiderGC) registerPodInformer() {
+	logger.Sugar().Infof("register pod informer")
+
```

Comment:
clientSet, err := kubernetes.NewForConfig(s.controllerMgr.GetConfig())
 		if nil != err {
 			logger.Sugar().Errorf("failed to init k8s clientSet, error: %v", err)
 		}